src/mpyl/cli/projects.py

```diff
@@ -99,24 +100,45 @@ def show_project(obj, name):
     )
 
 
-@projects.command(help="Validate the yaml of found projects against their schema")
+@projects.command(help="Validate the yaml of changed projects against their schema")
+@click.option(
+    "--all",
+    "all_",
+    is_flag=True,
+    help="Validate all project yaml's, regardless of changes on branch",
```

Comment:
Do we need to add default=False, or is that implied already?

Reply:
I copied it from the `build run` cmd and there we don't have a default?

Reply:
Should be fine then